src/components/QuranReader/index.tsx

```diff
@@ -133,7 +131,7 @@ const QuranReader = ({
             loader={
               <div className={styles.loadMoreContainer} key={0}>
                 <button type="button" onClick={loadMore} disabled={isValidating}>
```

Comment:
quick question, a bit unrelated to the changes in this PR.

is there a reason we make this a button?

Reply:
+1, I'd imagine that the spinner would not be interactive.